src/RAM/Arm/Arm.tsx

```diff
@@ -47,17 +47,12 @@ function arm(): void {
     moveArm = true
```

Comment:
Can the elbow only bend and twist at the same time?

Reply:
The && is only checking that the button bindings exist. The arm does not need to be receiving input for both movements in order for `moveArm` to be True